pkg/skaffold/build/gcb/cloud_build.go

```diff
@@ -253,9 +254,18 @@ func (b *Builder) createBucketIfNotExists(ctx context.Context, projectID, bucket
 		return errors.Wrapf(err, "getting bucket %s", bucket)
 	}
 
-	if err := c.Bucket(bucket).Create(ctx, projectID, &cstorage.BucketAttrs{
+	err = c.Bucket(bucket).Create(ctx, projectID, &cstorage.BucketAttrs{
 		Name: bucket,
-	}); err != nil {
+	})
+	if e, ok := err.(*googleapi.Error); ok {
+		if e.Code == http.StatusConflict {
+			// 409 errors are ok, there could have been a race condition or eventual consistency.
+			logrus.Debugf("Not creating bucket, got a 409 error indicating it already exists.")
```

Comment:
I'd probably log the err here for diagnostics (i.e. in case some other 409 is returned in the future)